rdr_client/work_queue.py

```diff
@@ -1,18 +1,50 @@
-""" Uses the service account 'awardee-pitt' to get participant summaries for work queue api
-    with an awardee of PITT"""
+""" Uses a service account to get and sync participant summaries for work queue api
+    You can only return results for the same awardee that the service account is connected to.
+
+    Run this using ./run_client.sh --project all-of-us-rdr-prod --account [your account]
+    --service_account [service account]@all-of-us-rdr-prod.iam.gserviceaccount.com work_queue.py
+
+    You should call code like this with credentials for a service account that has been set up for
+    your awardee by AllOfUs.
+    Expect to get the same participant summaries back in multiple responses (anything modified
+    within 60 seconds of the last summary in the previous response).
+    you'll get back relation="next" on the link if there are more results available in the RDR
+    that weren't returned and relation="sync" if there were.
+    Use the _count parameter to adjust this number. The default is 100
```

Comment:
Let's set _count to 5000 in the requests below, and update this comment. That's what we want them to use.

Reply:
done